acquia_lift_profiles/includes/acquia_lift_profiles.classes.inc

```diff
@@ -332,10 +345,11 @@ class ALProfilesAPI {
     // First get our Authorization header.
     $headers = array('Accept' => 'application/json');
     $url = $this->generateEndpoint('events/' . $event_name);
-    $auth_header = $this->getAuthHeader('PUT', $url, array('label' => $event_label, 'type' => $event_type), $headers);
+
+    $auth_header = $this->getAuthHeader('PUT', $url, array('customerSite' => $this->customerSite, 'label' => $event_label, 'type' => $event_type), $headers);
     $headers += array('Authorization' => $auth_header);
 
-    $response = $this->httpClient()->put($url . '?label=' . rawurlencode($event_label) . '&type=' . $event_type, $headers);
+    $response = $this->httpClient()->put($url . '?customerSite=' . $this->customerSite . '&label=' . rawurlencode($event_label) . '&type=' . $event_type, $headers);
```

Comment:
If the $this->customerSite is an empty string then omit the customerSite parameter in the url